homeassistant/components/hassio.py

```diff
@@ -0,0 +1,262 @@
+"""
+Exposes regular rest commands as services.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/hassio/
+"""
+import asyncio
+import logging
+import os
+
+import aiohttp
+from aiohttp import web
+from aiohttp.web_exceptions import HTTPBadGateway
+import async_timeout
+import voluptuous as vol
+
+from homeassistant.config import load_yaml_config_file
+from homeassistant.components.http import HomeAssistantView
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+
+DOMAIN = 'hassio'
+DEPENDENCIES = ['http']
+
+_LOGGER = logging.getLogger(__name__)
+
+TIMEOUT = 900
+
+SERVICE_HOST_SHUTDOWN = 'host_shutdown'
+SERVICE_HOST_REBOOT = 'host_reboot'
+
+SERVICE_HOST_UPDATE = 'host_update'
+SERVICE_SUPERVISOR_UPDATE = 'supervisor_update'
+SERVICE_HOMEASSISTANT_UPDATE = 'homeassistant_update'
+
+SERVICE_ADDON_INSTALL = 'addon_install'
+SERVICE_ADDON_UNINSTALL = 'addon_uninstall'
+SERVICE_ADDON_UPDATE = 'addon_update'
+SERVICE_ADDON_START = 'addon_start'
+SERVICE_ADDON_STOP = 'addon_stop'
+
+ATTR_ADDON = 'addon'
+ATTR_VERSION = 'version'
+
+
+SCHEMA_SERVICE_UPDATE = vol.Schema({
+    vol.Optional(ATTR_VERSION): cv.string,
+})
+
+SCHEMA_SERVICE_ADDONS = vol.Schema({
+    vol.Required(ATTR_ADDON): cv.slug,
+})
+
+SCHEMA_SERVICE_ADDONS_VERSION = SCHEMA_SERVICE_ADDONS.extend({
+    vol.Optional(ATTR_VERSION): cv.string,
+})
+
+
+SERVICE_MAP = {
```

Comment:
I think that we should not have services at all. We have an API, that should be enough?